man/systemd-firstboot.xml

```diff
@@ -34,12 +34,19 @@
   <refsect1>
     <title>Description</title>
 
-    <para><command>systemd-firstboot</command> initializes basic system settings interactively during the
-    first boot, or non-interactively on an offline system image. The service is started during boot if
-    <varname>ConditionFirstBoot=yes</varname> is met, which essentially means that <filename>/etc/</filename>
-    is unpopulated, see
+    <para>The <command>systemd-firstboot.service</command> is one of the units which are used to initialize
+    the machine during "First Boot", i.e. when the system is freshly installed or after a factory reset. The
+    <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry> manager
+    itself will perform some automatic initialization, and later
```

Comment:
"perform some automatic initialization" is a bit vague. I mean, I understand you mean setting of /etc/machine-id. I'd actually say it like that.

Also, "initialization" can mean anything, I'd not automatically assume this is about configuration files, but here it is. Hence, the "configuration initialization" would be better I think. I mean, of course systemd initializes itself and the kernel, by loading unit files and stuff, by bumping files in /proc/sys/ and so on, but that kind of "initialization" is not what's mean here, is it?